TicTacToe.Tests/CLITests.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace TicTacToe.Tests
+{
+    public class CLITests
+    {
+
+        [Fact]
+        public void TestLogToConsole()
+        {
+            var cli = new CLI();
+            string message = "test message";
+            var currentConsoleOut = Console.Out;
+            using (var consoleOutput = new ConsoleOutput())
+            {
+              cli.LogToConsole(message);
+              Assert.Equal(message, consoleOutput.GetOutput());
+            }
+        }
+
+        [Theory]
+        [InlineData("       |   X   |     ")]
+        [InlineData("  - - -+- - - -+- - -")]
+        [InlineData("       |       |   O ")]
+        public void TestPrintCurrentBoard(string line)

```

Comment:
I'm not that familiar with C#, but just wondering why this inlineData isnt showing / asserting against the full board? 

![Screen Shot 2020-02-24 at 4 59 46 PM](https://user-images.githubusercontent.com/29721784/75204910-15d76480-5727-11ea-95bb-e80f5fca51ab.png)

Reply:
I was only asserting against the first three lines of the board for that test, but it's a good idea to add the last two as well.